README.md

```diff
@@ -366,5 +389,60 @@ To use it:
 
 1. download [Postman](https://www.postman.com/) (or use their web client)
 2. import this collection
-3. set up a `publish_token` Postman environment variable like in the [docs](https://learning.postman.com/docs/sending-requests/variables/#variable-scopes)
+3. set up a `publish_token` Postman environment variable like in the
+[docs](https://learning.postman.com/docs/sending-requests/variables/#variable-scopes)
 4. send stuff to Zenodo by clicking buttons in Postman!
+
+## Supporting PUDL
+
+These raw data archivers are part of the Public Utility Data Liberation Project (PUDL),
+a project of [Catalyst Cooperative](https://catalyst.coop). PUDL has been made possible
+by the generous support of our sustainers, grant funders, and volunteer open source
+contributors.
+
+If you would like to support the ongoing development of PUDL, please consider
+[becoming a sustainer](https://opencollective.com/pudl).
```

Comment:
I don't love the idea of adding this full content below with all of the tiers and sustainers that we will certainly occasionally forget to update in multiple places. My suggestion would be to just add the opening paragraph here but not include all of the logos and details here.

Reply:
I wasn't thrilled with it either but I added it because we said we'd highlight them, and because I think it's also positive for us to show who else is involved in the project. But you're right that this repo gets far less traffic so it probably doesn't matter much. I'll remove.